scoresly.css

```diff
@@ -0,0 +1,90 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.post-credits {
+  display: flex;
+  justify-content: space-between;
+  text-wrap: wrap;
+}
+
+.left-credits, .right-credits {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin: 10px;
+}
+
+.right-credits {
+  text-align: end;
+}
+
+.left-credits{
+  width: 50vw;
```

Comment:
Either set this to 50% (which is what you actually mean in this case), or remove it. I know it causes the creators to look a bit weird, but I think we can redesign them as link a bit later.